packages/react/package.json

```diff
@@ -44,6 +49,7 @@
   "dependencies": {
     "@0no-co/graphql.web": "^1.0.4",
     "@gadgetinc/api-client-core": "^0.15.38",
+    "@gadgetinc/react": "workspace:^",
```

Comment:
This is gonna break stuff -- the package can't depend on itself, and can't depend on a workspace version of the package as that won't be available on npm. imports within the package should be relative to avoid referring to it by name